### PR TITLE
Add International Mobile Equipment Identity (IMEI)

### PR DESCRIPTION
**Pull request by acfreitas**
Removed from other PR #574 and create a new PR for Provider IMEI

### CODE REVIEW

**Comment by ronanguilloux** [comment]
I @acfreitas, thanks a lot for your contribution!

IMEI generation/validation uses Luhn check algorithm: I suggest to use the  `src/Faker/Calculator/Luhn` calcultor instead.

Unit tests should be appreciated too!

**Comment by acfreitas** [comment]
@ronanguilloux, I created the tests and edited the code to use ```src/Faker/Calculator/Luhn```. Thanks for your suggestion.

File: src/Faker/Provider/Imei.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Faker\Provider;
+
+use Faker\Calculator\Luhn;
+
+class Imei extends Base
```
Comment (fzaninotto) [inline src/Faker/Provider/Imei.php]: I suggest you move that to the PhoneNumber provider

Comment (acfreitas) [inline src/Faker/Provider/Imei.php]: @fzaninotto. the IMEI is used in some mobile devices, not just Phone. Do you think that move to PhoneNumber Provider is necessary?

Comment (fzaninotto) [inline src/Faker/Provider/Imei.php]: Let's consider that `PhoneNumber` is named `Phone` (which should have been the case from the beginning), so yes, this is necessary.

**Comment by fzaninotto** [comment]
Merged using the command line in 54e105fedd5715fb8070afb88334f212b6f2a84a. Thanks!

**Pull request closed without merging**